src/utils.jl

```diff
@@ -40,7 +40,7 @@ function create_LHS_matrix(z, λ, Q, G, h, A=nothing)::AbstractMatrix{Float64}
                 A         spzeros(p, m)          spzeros(p, p)]
     end
 end
-
+# TODO: this is the transpose, check back for usage
```

Comment:
what does this mean here?

Reply:
This comment is about the above function.
It is used in both back and forward methods.
However, it seems to be correct for backward methods, but it seems to be transposed in the forward method.